synapse/datamodel.py

```diff
@@ -109,8 +109,11 @@ def __init__(self, modl, form, name, typedef, info):
             self.isext = name.startswith('._')
         self.isform = False     # for quick Prop()/Form() detection
 
-        self.setperm = ('node', 'prop', 'set', self.full)
         self.delperm = ('node', 'prop', 'del', self.full)
```

Comment:
It feels like the compliment to `node.prop.set` getting two levels is `node.prop.del` also getting two levels. Do we want to knock that out at once so we have a consistent set of property setting / deleting permissions?